### PR TITLE
Ios bitcode spm

### PR DESCRIPTION
**Pull request by albho**

### CODE REVIEW

**Comment by albho** [comment]
@matt200-ok assigned you so you could also take a look at BrowserStack-related changes

File: binding/ios/CobraAppTest/PerformanceTest/PerformanceTest.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ class PerformanceTest: XCTestCase {
     }
 
     func testPerformance() throws {
-        try XCTSkipIf(thresholdString == "{PERFORMANCE_THRESHOLD_SEC}")
+        try XCTSkipIf(thresholdString == nil)
```
Comment (laves) [inline binding/ios/CobraAppTest/PerformanceTest/PerformanceTest.swift:26]: I think this was set to `{PERFORMANCE_THRESHOLD_SEC}` so that if you didn't replace the `thresholdString` default value, it would skip the test. I think we still need that otherwise it will run the test by default unless you explicitly set it to nil.

**Review by matt200-ok — APPROVED** [review]
All the BrowserStack changes look good

**Review by laves — APPROVED** [review]
Have you tested to see if it reports test failures accurately? Like intentionally make a test fail and ensure you get a useful report back

File: binding/ios/CobraAppTest/PerformanceTest/PerformanceTest.swift (hunk 1 above)
Comment (albho) [reply binding/ios/CobraAppTest/PerformanceTest/PerformanceTest.swift]: I reverted this, although I'm a bit confused - the `Inject Performance Threshold` step in the actions seemed to replace all instances of `{PERFORMANCE_THRESHOLD_SEC}`, causing the test to be skipped - not sure how this passed in the past. I changed it though to only replace the first instance.

**Comment by albho** [comment]
> Have you tested to see if it reports test failures accurately? Like intentionally make a test fail and ensure you get a useful report back

@laves Tested this now, and yes we do (tested perf threshold of 0 and invalid access key)
